frontend/gpay_agent/lib/datamodels/agent.dart

```diff
@@ -1,17 +1,27 @@
 ///Model class for storing agent details.
 class Agent{
-  int AgentID;
+  String AgentEmail;
+  String AgentCreationDateTime;   //Look into the datatype
```

Comment:
Use Timestamp used by vahini?

Reply:
Time stamp not being used in agent front end app, so leaving it as String